New commit! I think I fixed the compiling error that is below.
```
Run svn checkout -q https://github.com/arichorn/sdks/trunk/iPhoneOS16.2.sdk
svn: E170013: Unable to connect to a repository at URL 'https://github.com/arichorn/sdks/trunk/iPhoneOS16.2.sdk'
svn: E900002: Subversion support temporarily disabled.

GitHub has temporarily disabled Subversion support as part of a planned brownout
in order to give you advance notice of the upcoming permanent removal of
Subversion support on 2024-01-08.

Please see https://github.blog/2023-01-20-sunsetting-subversion-support/ for details.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/buildapp.yml
+++ b/.github/workflows/buildapp.yml
@@ -77,6 +77,9 @@ jobs:
           cd sdks
           git sparse-checkout set --no-cone iPhoneOS${{ inputs.sdk_version }}.sdk
           git checkout
+          mv *.sdk $THEOS/sdks
+        env:
+          THEOS: ${{ github.workspace }}/theos
 
       - name: Setup Theos Jailed
         uses: actions/checkout@v4

--- a/.github/workflows/buildapp.yml
+++ b/.github/workflows/buildapp.yml
@@ -21,6 +21,11 @@ on:
         default: ""
         required: true
         type: string
+      sdk_version:
+        description: "iOS SDK version to be used during build"
+        default: "16.2"
+        required: true
+        type: string
       create_release:
         description: "Create a draft release"
         default: true
@@ -40,7 +45,7 @@ jobs:
 
     steps:
       - name: Checkout Main
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           path: main
           submodules: recursive
@@ -49,7 +54,7 @@ jobs:
         run: brew install ldid dpkg
 
       - name: Setup Theos
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           repository: theos/theos
           ref: 9941262c450c0abac524574cb96b9a4cdf5e50f4
@@ -60,19 +65,21 @@ jobs:
         id: SDK
         uses: actions/cache@v3.3.2
         env:
-          cache-name: iOS-16.2-SDK
+          cache-name: iOS-${{ inputs.sdk_version }}-SDK
         with:
           path: theos/sdks/
           key: ${{ env.cache-name }}
 
-      - name: Download iOS 16.2 SDK
+      - name: Download iOS SDK
         if: steps.SDK.outputs.cache-hit != 'true'
         run: |
-          git clone --quiet https://github.com/arichorn/sdks.git
-          mv sdks/iPhoneOS16.2.sdk $THEOS/sdks
+          git clone -n --depth=1 --filter=tree:0 https://github.com/arichorn/sdks/
+          cd sdks
+          git sparse-checkout set --no-cone iPhoneOS${{ inputs.sdk_version }}.sdk
+          git checkout
 
       - name: Setup Theos Jailed
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           repository: qnblackcat/theos-jailed
           ref: master

